vaas/vaas/metrics/prometheus.py

```diff
@@ -0,0 +1,45 @@
+from datetime import timedelta
+from typing import Optional, TypedDict, Union
+from django.conf import settings
+from prometheus_client import CollectorRegistry, Gauge, Summary, push_to_gateway
+
+from vaas.metrics.models import Metrics
+
+
+class MetricsBucket(TypedDict, total=False):
+    name: str
+    metric: Union[Summary, Gauge]
+
+
+class PrometheusClient:
+    def __init__(self) -> None:
+        self.host: str = f'{settings.PROMETHEUS_GATEWAY_HOST}:{settings.PROMETHEUS_GATEWAY_PORT}'
+        self.job: str = 'vaas'
```

Comment:
Shouldn't it be passed via CONFIG?

Reply:
Yes. I have already this variable in settings but forgot to use it here. 
Nice catch!